src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -296,7 +279,7 @@ public FormulaValue GetResult(IRContext irContext)
             }
         }
 
-        private class MaxDateAgg : IAggregator
+        private class MaxDateAndDateTimeAgg : IAggregator
```

Comment:
>MaxDateAndDateTimeAgg [](http://example.com/codeflow?start=22&length=21)

Merged the aggregator for date and datetime values, since the logic (and underlying data type, DateTime) is the same.